src/libinjector/injector.c

```diff
@@ -537,7 +537,6 @@ bool setup_stack_64(
 }
 
 void init_argument(
```

Comment:
I think it would make sense to just convert this function into a macro.

Reply:
@tklengyel do You suggest this for optimization? If so I have made this function `inline` for [these](https://isocpp.org/wiki/faq/inline-functions#inline-vs-macros) reason.